autoload/slime.vim

```diff
@@ -348,7 +348,13 @@ function! slime#send(text)
   " so we, possibly, send many strings -- but probably just one
   let pieces = s:_EscapeText(a:text)
   for piece in pieces
-    call s:SlimeDispatch('Send', b:slime_config, piece)
+    if type(piece) == 0
+      if piece != 0
```

Comment:
would that be better as `piece > 0`?   🤔

Reply:
yeah… there's no "great" way to flag these things

I think it's ok to ignore negative and zero values.

Reply:
OK, I'll change it.